documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -16,4 +16,8 @@
 ### Resolved issues
 * (IDETECT-4056) Resolved an issue where no components were reported by CPAN detector.
   If the cpan command has not been previously configured and run on the system, [solution_name] instructs CPAN to accept default configurations.
-* (IDETECT-3843) Additional information is now provided when [solution_name] fails to update and [solution_name] is internally hosted.
\ No newline at end of file
+* (IDETECT-3843) Additional information is now provided when [solution_name] fails to update and [solution_name] is internally hosted.
+
+### Dependency updates
+* Upgraded [solution_name] Alpine Docker image to 3.18 to pull the latest curl version with no known vulnerabilities.
```

Comment:
@niravrsynopsys For Alpine, can you specify images (both standard and buildless) to be more clear. cc: @sig-cpotts